opm/simulators/wells/WellGroupHelpers.cpp

```diff
@@ -963,7 +963,7 @@ computeNetworkPressures(const Network::ExtNetwork& network,
 
                     if (well.isInjector() || !well_state.isOpen(wellname)) continue;
                     const Scalar efficiency = well.getEfficiencyFactor(/*network*/ true) * well_state.getGlobalEfficiencyScalingFactor(wellname);
-                    node_inflows[node][BlackoilPhases::Vapour] += well_state.getALQ(wellname) * efficiency;
+                    node_inflows[node][BlackoilPhases::Vapour] += well_state.well(wellname).alq_state.get() * efficiency;
                 }
```

Comment:
I assumed this would not work in parallel, but Jenkins seems to be happy. Do we have a test case that exposes this feature? I.e. gaslift in network.

Reply:
It does not look that we have in the regression tests or parallel tests, but we might have some examples in the opm-tests. Maybe it is a good time if you can add one based on one of them.

Reply:
Do you know which test is relevant?

Reply:
I thought you work with gas lift more, you might know more? I have not tested much. If it is not relevant cases in opm-tests, maybe you can add one from your development?

Reply:
I just tested with NETWORK-01_STANDARD where I added 'FLO' for item 6. I.e. with the latest change it should also work in parallel for item6 = FLO.